libs/StatusGoQt/src/StatusGo/Messages/MessagesApi.cpp

```diff
@@ -0,0 +1,21 @@
+#include "MessagesApi.h"
+#include "Metadata/api_response.h"
+#include "InputMessage.h"
+#include "Utils.h"
+
+#include <Helpers/conversions.h>
+
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+using namespace Status::StatusGo;
+
+void Messages::sendMessage(const InputMessage &message)
```

Comment:
Would be great to test this API by extending `test/libs/StatusGoQt/test_messaging.cpp`

Reply:
@endulab end-to-end testing of this API is not straightforward. However, just a simple call that will validate RPC call ant that the message reached the waku library endpoint should be easy to implement